runtime/src/Runtime.cpp

```diff
@@ -200,20 +201,22 @@ namespace RType::Runtime
             }
             return this->loadPrefab(path);
         });
-        m_lua.set_function("playSound", [&](RType::Runtime::ECS::Entity e, const char *path) -> void {
-            if (isServer())
-                return;
+        m_lua.set_function(
```

Comment:
Playsound with three args ? Warning for old script ! Update this or update old script